app/src/main/res/layout/checklists_fragment.xml

```diff
@@ -13,40 +13,55 @@
     </data>
 
     <androidx.constraintlayout.widget.ConstraintLayout
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            tools:context=".presentation.checklists.ChecklistsFragment">
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        tools:context=".presentation.checklists.ChecklistsFragment">
+
+        <ImageView
```

Comment:
There should be a centered button instead of a plus icon.

Reply:
Done